app/components/navigation_header/header.tsx

```diff
@@ -57,6 +57,7 @@ const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => ({
         lineHeight: 12,
         marginBottom: 8,
         marginTop: 2,
+        height: 13,
```

Comment:
what is this and why is it needed ?

Reply:
This is a small unrelated fix. The text with "x members >" shown in the header is slightly cropped below. A pixel higher makes it look ok.